src/by/epam/ayem/module5/model/TreasureHouse.java

```diff
@@ -0,0 +1,35 @@
+package by.epam.ayem.module5.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/*Задача 4.
+Создать консольное приложение, удовлетворяющее следующим требованиям:
+1) Приложение должно быть объектно-, а не процедурно-ориентированным.
+2) Каждый класс должен иметь отражающее смысл название и информативный состав.
+3) Наследование должно применяться только тогда, когда это имеет смысл.
+4) При кодировании должны быть использованы соглашения об оформления кода java code convention.
+5) Классы должны быть грамотно разложены по пакетам.
+7) Консольное меню должно быть минимальным.
+8) Для хранения данных можно использовать файлы.
+
+Дракон и его сокровища. Создать программу, позволяющую обрабатывать сведения о 100 сокровищах в пещере дракона.
+Реализовать возможность просмотра сокровищ, выбора самого дорогого по смоимости сокровища и выбора сокровищ
+на заданную сумму.*/
+
+public class TreasureHouse {
```

Comment:
Why House? May be Dungeon?

Reply:
It's kind of a new generation dragon. It keeps its treasure in a treasure house in Switzerland.